src/System.Time/src/System/Date.cs

```diff
@@ -0,0 +1,1629 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.Diagnostics.Contracts;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+
+namespace System
```

Comment:
These types complement the existing types.  They don't replace them.  IMHO, it wouldn't make sense to find them in a new namespace unless there were a complete set of new types there.  We should discuss further.